frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -172,6 +172,37 @@ function addPercentSignsToDisplayNames(series) {
   );
 }
 
+// Store a "decimals" property on the column that is normalized
+function addDecimalsToPercentColumn(series, decimals) {
+  return series.map(s =>
+    updateIn(s, ["data", "cols", 1], col => ({ ...col, decimals })),
```

Comment:
btw I think `assicIn(s, ["data", "cols", 1, "decimals"], decimals)` is equivalent.